src/EventStore.Core/Services/Transport/Grpc/Cluster.Elections.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using EventStore.Client;
+using EventStore.Client.Shared;
+using EventStore.Cluster;
+using EventStore.Core.Messages;
+using Grpc.Core;
+
+namespace EventStore.Core.Services.Transport.Grpc {
+	partial class Cluster {
+		public override Task<Empty> ViewChange(ViewChangeRequest request, ServerCallContext context) {
```

Comment:
We can cache the completed task in this case as a static singleton which will reduce allocations